x/params/keeper_test.go

```diff
@@ -216,3 +216,38 @@ func TestSubspace(t *testing.T) {
 		require.Equal(t, kv.param, indirect(kv.ptr), "stored param not equal, tc #%d", i)
 	}
 }
+
+type paramJSON struct {
+	Param1 int64  `json:"param1,omitempty"`
+	Param2 string `json:"param2,omitempty"`
+}
+
+func TestJSONUpdate(t *testing.T) {
+	cdc := createTestCodec()
+	mkey := sdk.NewKVStoreKey("test")
```

Comment:
can we create a `test_common.go`  file and move this there? Check the TestInput functions from other modules